code/extensions/FileSecured.php

```diff
@@ -393,11 +393,13 @@ public static function find_or_make_secured($folderPath) {
     }
 
     /**
-     * 
+     * Gets us the module's root directory. Because this is created on-the-fly, then we can never accurately know
+     * its ID.
+     *
      * @return Folder
      */
     public static function getSecuredRoot() {
-        return $securedRoot = Folder::get_one("Folder", "\"ParentID\" = '0' AND \"Secured\" = '1'");
+        return DataObject::get_one("File", "\"ParentID\" = '0' AND \"Secured\" = '1'");
```

Comment:
don't know why the old query doesn't work, cos the old one sounds more correct than the new one, because we definitely want to get that "_securedassets" folder object.
But I believe you, so don't mind, as long as the change works for you. :)